web/client/components/I18N/FlagButton.jsx

```diff
@@ -13,10 +13,11 @@ import OverlayTrigger from '../misc/OverlayTrigger';
 import { getSupportedLocales } from '../../utils/LocaleUtils';
 import Button from '../misc/Button';
 
-class LangBar extends React.Component {
+class FlagButton extends React.Component {
```

Comment:
Can you explain with the changes in this component are related to "Removing warnings"?

Reply:
if you mean, the component name, i changed it because it was wrong and not aligned to filename